package.json

```diff
@@ -7,11 +7,14 @@
     "dev": "turbo run dev --parallel",
     "lint": "turbo run lint",
     "prepare": "husky install",
+    "release": "changeset version && pnpm install",
+    "publish": "changeset publish",
```

Comment:
From https://pnpm.io/using-changesets#releasing-changes, publish should just be  `pnpm publish -r`